package.json

```diff
@@ -17,14 +17,16 @@
     "packages:changed": "lerna changed",
     "packages:list": "lerna ls",
     "packages:test": "lerna run test",
-    "test": "npm-run-all --serial lint packages:test format",
+    "test": "npm-run-all --serial lint:* packages:test format",
```

Comment:
Oh, this also includes the `fix` task. Is it the intention?

Reply:
Thanks, fixed.